src/libsync/owncloudpropagator.cpp

```diff
@@ -1004,10 +1005,11 @@ void PropagateDirectory::slotSubJobsFinished(SyncFileItem::Status status)
         if (_item->_instruction == CSYNC_INSTRUCTION_RENAME
             || _item->_instruction == CSYNC_INSTRUCTION_NEW
             || _item->_instruction == CSYNC_INSTRUCTION_UPDATE_METADATA) {
-            if (!propagator()->updateMetadata(*_item)) {
+            const auto result = propagator()->updateMetadata(*_item);
+            if (!result) {
                 status = _item->_status = SyncFileItem::FatalError;
-                _item->_errorString = tr("Error writing metadata to the database");
-                qCWarning(lcDirectory) << "Error writing to the database for file" << _item->_file;
+                _item->_errorString = tr("Error updating metadata: %1").arg(result.error());
+                qCWarning(lcDirectory) << "Error writing to the database for file" << _item->_file << "with" << result.error();
```

Comment:
```suggestion
                qCWarning(lcDirectory) << "Error updating metadata for file" << _item->_file << "with" << result.error();
```

qCWarning and item->_errorString should match.